config/environments/production.rb

```diff
@@ -58,11 +58,6 @@
   # Prepend all log lines with the following tags.
   config.log_tags = [:request_id]
 
-  # Info include generic and useful information about system operation, but avoids logging too much
-  # information to avoid inadvertent exposure of personally identifiable information (PII). If you
-  # want to log everything, set the level to "debug".
-  config.log_level = ENV.fetch("RAILS_LOG_LEVEL", "info")
-
```

Comment:
I moved this up to application while trying to look at logging in my dev env. Is there a reason to not have it there though? wanted to make sure before this gets merged

Reply:
In most situations, there are intentionally different log levels in the different environments. The default log_level is debug which works great for dev and test so it's just production which overrides this and sets it to info.

Reply:
ah ok. I'm still slowly remembering rails. I was having trouble with logging, but I think it was due to something else. I'll change this back